sources/Services/Audio/AudioOutDriver.cpp

```diff
@@ -80,34 +90,91 @@ void AudioOutDriver::clipToMix() {
 		fixed *p=primarySoundBuffer_ ;
         
 		fixed v;
-		fixed f_32767=i2fp(32767) ;
-		fixed f_m32768=i2fp(-32768) ;
+		fixed f_32767=i2fp(32767);
+		fixed f_m32768=i2fp(-32768);
+		fixed fl_32767=fp2fl(f_32767);
+		fixed fl_m32768=fp2fl(f_m32768);
+
+    float x;
```

Comment:
I've been going through this section a few times now. I guess it's a standard cubic symmetric clipping algorithm? In that case, could this variable have a more descriptive name such as clippedVal? =)

Reply:
Tried it, but I think it gets confusing, since it's not really clipped half of the time. Added a link to the algorithm description instead, it does make sense in context